src/prefect/client/client.py

```diff
@@ -1658,6 +1659,9 @@ def get_flow_run_state(self, flow_run_id: str) -> "prefect.engine.state.State":
 
         flow_run = self.graphql(query).data.flow_run_by_pk
 
+        if not flow_run:
```

Comment:
just a nit: do we intend this to fail if the flow_run is explicitly `None`?

Reply:
I'm intentionally covering any form empty data here as the return line will fail with an `AttributeError`